docs/docs/recipes-pages-layouts.md

```diff
@@ -0,0 +1,233 @@
+---
+title: Pages and Layouts
```

Comment:
Same as the previous page: can we combine titles and adjust the heading structure so the text isn't repeated?